NGRValidator/NGRValidator/NGRPropertyValidator.h

```diff
@@ -22,58 +22,58 @@ extern NSUInteger const NGRPropertyValidatorDefaultPriority;
  *  @param property Name of property given as NSString.
  *  @return Instance of NGPropertyValidator used to specify next validation rules.
  */
-- (instancetype)initWithProperty:(NSString *)property NS_DESIGNATED_INITIALIZER;
-- (instancetype)init NS_UNAVAILABLE;
+- (nullable instancetype)initWithProperty:(nullable NSString *)property NS_DESIGNATED_INITIALIZER;
+- (nullable instancetype)init NS_UNAVAILABLE;
 
 /**
  *  Name of validated property.
  */
-@property (strong, nonatomic, readonly) NSString *property;
+@property (strong, nonatomic, readonly, nonnull) NSString *property;
```

Comment:
It'd be better to wrap this header in `NS_ASSUME_NONNULL` macros and remove `nonnull` annotations.